docs/sources/datasources/tempo/_index.md

```diff
@@ -20,11 +20,11 @@ weight: 1400
 
 # Tempo data source
 
-Grafana ships with built-in support for [Tempo](https://grafana.com/docs/tempo/latest/), a high-volume, minimal-dependency trace storage, open-source tracing solution from Grafana Labs. This topic explains configuration and queries specific to the Tempo data source.
+Grafana ships with built-in support for [Tempo](https://grafana.com/docs/tempo/latest/), a high-volume, minimal-dependency trace storage, open source tracing solution from Grafana Labs. This topic explains configuration and queries specific to the Tempo data source.
 
 For instructions on how to add a data source to Grafana, refer to the [administration documentation][data-source-management].
 Only users with the organization administrator role can add data sources.
-Administrators can also [configure the data source via YAML](#provision-the-data-source) with Grafana's provisioning system.
+Administrators can also [configure the data source via YAML](configure-tempo-data-source#provision-the-data-source) with Grafana's provisioning system.
```

Comment:
Can this be replaced with a full URL as per https://grafana.com/docs/writers-toolkit/write/links/#link-to-grafanacom-pages? I think it makes it easier to read and understand for other writers and contributors.

Reply:
Ah yeah, very interesting point. In that case, the guidance is to use `docs/reference` https://grafana.com/docs/writers-toolkit/write/links/#link-from-source-content-thats-used-in-multiple-projects

Reply:
Ah, looks like I have to add a reference for the docs/reference short code usage.